tests/cpp/test_resharding.cpp

```diff
@@ -419,17 +597,17 @@ TEST_P(ReshardingTest, Insert) {
   fusion_->addOutput(tv1);
   fusion_->addOutput(tv5);
 
-  if (is_tv0_tv3_tv5_sharded) {
-    tv0->axis(sharded_axis)->parallelize(ParallelType::DIDx);
-    tv3->axis(sharded_axis)->parallelize(ParallelType::DIDx);
-    tv5->axis(sharded_axis)->parallelize(ParallelType::DIDx);
+  if (is_tv0_tv5_sharded) {
+    tv0->axis(kShardedAxis)->parallelize(ParallelType::DIDx);
+    // tv3->axis(kShardedAxis) is a reduction, so don't shard it.
```

Comment:
An incidental cleanup per https://github.com/NVIDIA/Fuser/blob/3266b9d21cb82272fe6e766b71fb9a9f298de833/csrc/preseg_passes/propagate_shardings.cpp#L101-L140. I'm fine keeping the DIDx -- it doesn't hurt.